lib/permissions.js

```diff
@@ -1,60 +1,18 @@
 'use strict';
 
-const request = require('./utils/request');
-const R = require('ramda');
-
-const isPermission = (item) => R.is(Array, item) && item.length === 3 && R.is(String, item[0]) && R.is(String, item[1]);
-
-function parsePermissions (items) {
-  return items.reduce((results, item) => {
-    if (isPermission(item)) {
-      return R.append({
-        permission: item[0],
-        description: item[1]
-      }, results);
-    }
-    if (R.is(Array, item)) {
-      return results.concat(parsePermissions(item));
-    }
-    return results;
-  }, []);
-}
+const { processPermissions } = require('./requesters/permissionMappedRequests');
```

Comment:
Why is most of the logic in this separate file, if this one is almost empty?

Reply:
@facundoolano 
I tried to follow a "pattern", leaving only the initial request in the base files.
So in the end is basically
 - root files for methods: initial request + validation (what can be separated as well)
 - requests in the /requesters folder
 - parsers in the /utils folder (the can be changed to parsers folder, for better "readability")